index.js

```diff
@@ -198,19 +198,26 @@ class ClientServiceBase {
   /**
    * @param {number} contractId
    * @param {Object} argument
+   * @param {Object} [functionArgument=null]
    * @return {Promise<ClientServiceResponse|void|*>}
    */
-  async executeContract(contractId, argument) {
+  async executeContract(contractId, argument, functionArgument) {
     argument['nonce'] = new Date().getTime().toString();
     const argumentJson = JSON.stringify(argument);
 
-    const request = new ContractExecutionRequestBuilder(
+    const builder = new ContractExecutionRequestBuilder(
         new this.protobuf.ContractExecutionRequest(), this.signer)
         .withContractId(contractId)
         .withContractArgument(argumentJson)
         .withCertHolderId(this.certHolderId)
-        .withCertVersion(this.certVersion)
-        .build();
+        .withCertVersion(this.certVersion);
+
+    if (functionArgument != null) {
+      const functionArgumentJson = JSON.stringify(functionArgument);
```

Comment:
Just a question and suggestion.
`JSON.stringify(null)` seems return `null` so 
we might be able to just do like
```
.withCertVersion(this.certVersion)
.withFunctionArgument(JSON.stringify(functionArgumentJson))
.build();
```

and doing like this in the builder might be a little better ?
```
request.setCertVersion(this.certVersion);
if (this.functionArgument != null) {
  request.setFunctionArgument(this.functionArgument);
}
```
It won't be much different but I feel it is slightly better.
Please let me know your thought.